lib/diagnostic.rb

```diff
@@ -7,36 +6,36 @@ class Diagnostic
   # Assume each entry in the list implements a length method that returns
   # a Fixnum.
   def longest(list)
-    # Your code goes here
+    list.sort { |a, b| b.length <=> a.length }[0]
```

Comment:
This is super clever! Here's a quicker way we can do it though:

`list.max_by(&:length)`